TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -126,37 +123,62 @@ static class Parameters {
                 arity = 0,
                 description = "Do not check the server's certificate")
         boolean allowInsecureConnections;
+
+
         @Parameter(required = false,
-                names = {"--auth-header-value"},
+                names = {REMOVE_AUTH_HEADER_VALUE_ARG},
+                arity = 0,
+                description = "Remove the authorization header if present and do not replace it with anything.  " +
+                        "(cannot be used with other auth arguments)")
+        boolean removeAuthHeader;
+        @Parameter(required = false,
+                names = {AUTH_HEADER_VALUE_ARG},
                 arity = 1,
-                description = "Prepared value to use for the \"authorization\" header of each request")
+                description = "Static alue to use for the \"authorization\" header of each request " +
+                        "(cannot be used with other auth arguments)")
         String authHeaderValue;
         @Parameter(required = false,
-            names = {"--aws-auth-header-user"},
-            arity = 1,
-            description = "Plaintext username to use for the username section of a constructed \"authorization\" header for each request")
+                names = {AWS_AUTH_HEADER_USER_ARG},
+                arity = 1,
+                description = "Plaintext username to use for the username section of a constructed \"authorization\" " +
+                        "header for each request" +
```

Comment:
This sounds like a basic auth thing, but the param name has `aws` in it. Is this from teh secrets manager stuff? So this is incorporated with AWS, but doesn't have anything to do with SigV4?

Also, it says "cannot be used with other auth arguments", but my understanding (if I'm correctly guessing what it is) is that this _should_ be used with AWS_AUTH_HEADER_SECRET_ARG